gocourse07/birdaviary/internal/logger/log.go

```diff
@@ -0,0 +1,18 @@
+package logger
+
+import (
+	"io"
+	"log/slog"
+)
+
+type Logger struct {
+	*slog.Logger
+}
+
+func New(writer io.Writer) *Logger {
+	log := &Logger{
+		Logger: slog.New(slog.NewJSONHandler(writer, nil)),
+	}
+	slog.SetDefault(log.Logger)
+	return log
```

Comment:
Можна спростити
```suggestion
	return &Logger{
		Logger: slog.New(slog.NewJSONHandler(writer, nil)),
	}
```

`slog.SetDefault` не потрібен, тому що програма не використовує `slog.Info`, `slog.Warn` і т.д.